verl/workers/fsdp_workers.py

```diff
@@ -50,14 +50,37 @@
     load_fsdp_optimizer,
     offload_fsdp_model_to_cpu,
     offload_fsdp_optimizer,
+    layered_summon_lora_params,
 )
 from verl.utils.import_utils import import_external_libs
 from verl.utils.model import compute_position_id_with_mask
 from verl.workers.sharding_manager.fsdp_ulysses import FSDPUlyssesShardingManager
 
+from peft import LoraConfig, TaskType, get_peft_model
+from codetiming import Timer
+
+import torch.distributed as dist
+from torch.distributed.fsdp import FullyShardedDataParallel as FSDP
+from peft import PeftModel
+from peft.utils.save_and_load import get_peft_model_state_dict
+from safetensors.torch import save_file
+from dataclasses import asdict
+import json
+
+
 logger = logging.getLogger(__file__)
 logger.setLevel(os.getenv("VERL_LOGGING_LEVEL", "WARN"))
 
+def convert_to_regular_types(obj):
```

Comment:
Shall we make this a general util and put in under https://github.com/volcengine/verl/blob/main/verl/utils/py_functional.py